common/web/types/src/kpj/keyman-developer-project.ts

```diff
@@ -127,51 +127,38 @@ export class KeymanDeveloperProjectOptions {
   }
 };
 
-export type KeymanDeveloperProjectFile = KeymanDeveloperProjectFile10 | KeymanDeveloperProjectFile20;
-
-export class KeymanDeveloperProjectFile10 {
-  readonly id: string;           // 1.0 only
-  readonly filename: string;
+export interface KeymanDeveloperProjectFile {
+  get filename(): string;
+  get fileType(): string;
```

Comment:
Not that you actually need to change it, but...

```suggestion
  readonly filename: string;
  readonly fileType: string;
```

will still work.  It's compatible with property accessors like what you have at line 137.